NumLib/ODESolver/ODESystem.h

```diff
@@ -65,7 +65,8 @@ class ODESystem<Matrix, Vector,
 
     using Index = typename MathLib::MatrixVectorTraits<Matrix>::Index;
 
-    virtual std::vector<ProcessLib::DirichletBc<Index> > const* getKnownSolutions() const
+    virtual std::vector<ProcessLib::DirichletBc<Index>> const*
```

Comment:
Is it necessary to provide some doc? If so, could you please add a line of doxygen doc, please. I forgot that when writing this code.